### ISSUE TITLE
LDAP Authentication isn't working Anymore

### ISSUE THREAD

**Issue by DerP4si** [jellyfin/jellyfin#11195]
### Please describe your bug

Users that are authenticated via ldap aren't able to login anymore. 

### Reproduction Steps

1.Docker 10.9.0 - 2024031805
2. LDAP setup just as in 10.8.13
3. LDAP user can't login.

### Jellyfin Version

Unstable (master branch)

### Environment

```markdown
- Virtualization:Docker
- Clients:All
- Browser:All
- FFmpeg Version:ffmpeg6_6.0.1-4
- Hardware Acceleration:VAAPI
- GPU Model:Vega 8
- Plugins:LDAP
- Reverse Proxy:traefik
- Storage:CIFS
```

### Jellyfin logs

```shell
[2024-03-21 13:55:36.761 +00:00] [WRN] User "muster" was found with invalid/missing Authentication Provider "Jellyfin.Plugin.LDAP_Auth.LdapAuthenticationProviderPlugin". Assigning user to InvalidAuthProvider until this is corrected
[2024-03-21 13:55:36.771 +00:00] [INF] Authentication request for "muster" has been denied (IP: "162.158.158.81").
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cvium** [comment]
Plugins aren't guaranteed to be compatible with unstable.

**Comment by DerP4si** [comment]
Ok... I just wanted to let you know, so if 10.9.0 arrives everything works as intended...

**Comment by disconn3ct** [comment]
10.9.0 is out and this is still broken.
```
[2024-05-12 08:50:24.558 -04:00] [INF] [33] Jellyfin.Server.Implementations.Users.UserManager: Authentication request for "tv" has been denied (IP: "10.1.1.1").
[2024-05-12 08:50:24.647 -04:00] [ERR] [33] Jellyfin.Api.Middleware.ExceptionMiddleware: Error processing request: "Invalid username or password entered". URL "POST" "/Users/AuthenticateByName".
[2024-05-12 08:52:04.928 -04:00] [WRN] [36] Jellyfin.Server.Implementations.Users.UserManager: User "tv" was found with invalid/missing Authentication Provider "Jellyfin.Plugin.LDAP_Auth.LdapAuthenticationProviderPlugin". Assigning user to InvalidAuthProvider until this is corrected
```

**Comment by robigan** [comment]
Same here, @disconn3ct How did you manually update it?

**Comment by pathetic-lynx** [comment]
@robigan in the admin settings, navigate to Plugins/Catalog/LDAP, then select the most recent version to install.

No idea why the update job did not catch this tho

**Comment by robigan** [comment]
> @robigan in the admin settings, navigate to Plugins/Catalog/LDAP, then select the most recent version to install.
> 
> 
> 
> No idea why the update job did not catch this tho

How can I update it from the dashboard if I can't login...

Anyways I did it my downloading the zip, uploading it to my machine and then unzipping it in the container

**Comment by DerP4si** [comment]
> @robigan in the admin settings, navigate to Plugins/Catalog/LDAP, then select the most recent version to install.
> 
> No idea why the update job did not catch this tho

Thanks!!!

**Comment by disconn3ct** [comment]
> if I can't login..

You should have a local admin account for things like "configuring authentication"

**Comment by robigan** [comment]
> > if I can't login..
> 
> You should have a local admin account for things like "configuring authentication"

Nope, a shell should always suffice. If I'm having problems with SSO, then I'm probably already beyond ducked in other ways. Although this time around it seems to be a Jellyfin bug

**Comment by detrout** [comment]
> No idea why the update job did not catch this tho

I had this LDAP authentication problem and wanted to say I updated using the jellyfin debian repository, in case the upgrade method might have something to with why the the plugin wasn't updated.

It'd be nice if the My Plugins view had a visual indication of what the latest version is, to make it easier to figure out the plugin might need to be updated

**Comment by jamerk** [comment]
I'm having this exact same issue but with Jellyfin running in a VM. I tried to upgrade my instance twice to JellyFin 10.9.5 but both times LDAP failed and kept giving me the "invalid/missing Authentication Provider" error (LDAP Plugin is version 18), so I restored from backup and cloned my production VM to try and sort this out. I upgraded it to 10.9.6 yesterday and authentication via LDAP still doesn't work. Has anyone had any luck figuring this out?

**Comment by crobibero** [comment]
v19 is the latest version of the LDAP plugin.

**Comment by jamerk** [comment]
> v19 is the latest version of the LDAP plugin.

This just fixed the issue in 10.9.6 for me, though version 19 of the plugin does not show up in the plugin catalog for me so I had to manually update to it. Thank you for your help @crobibero and for your work on this project
